docs/iris/src/whatsnew/contributions_2.2.0/newfeature_2018-Oct-05_contiguity-checking-and-masking.txt

```diff
@@ -0,0 +1,5 @@
+* Users can now check their cubes for any discontiguities in bounds arrays
```

Comment:
I don't like the use of "users" here. It implies that (a) there's an 'us' (devs) and 'them' (users), and not only that but (b) the whatsnew is by devs and for devs, and that users should be a little uncomfortable to also be here.

That said, it's hard to write these things! You can avoid the 'us' vs 'them' problem by using "you" in place of "users", which is more friendly, or not cast the whatsnew in terms of a person at all:

```
The function "find_discontiguities_in_bounds" can be used to check for discontiguities in the bounds arrays of cube coordinates.
Additionally, discontiguous points in coordinates can be explicitly masked using "mask_data_at_discontiguities".
```

Or something.

Reply:
Well, I didn't expect you to hear you promoting use of the passive voice @dkillick !

Not that I object to that, but I know other people do.